scripts/Imaging/imgpy/parallel/utility.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import (absolute_import, division, print_function)
+
+import numpy as np
+
+
+def create_shared_array(shape, dtype=np.float32):
+    import multiprocessing
+    import ctypes
```

Comment:
The problem then is that if we fail the import when trying to run something in parallel the whole execution will stop, while if it's contained within multiprocessing_available, it will not stop, but simply run the sequential code